### DISCUSSION TITLE
"Merge" two usernames

### DISCUSSION THREAD

**Discussion by BusyBeaver-42** [community/community#24020] [New to GitHub]
<p>I started using git before creating a GitHub account. Hence in Git Bash, user.name was set to my real name (I had forgotten to change it).</p>
<p>In a repo, I made commits from both Git Bash and Atom. As a result, I appear under two different names in the same repo.</p>
<p>Is there a way to change that ? (I would like to appear only under my GitHub username)</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>All you need to do to have your commits associated with your GitHub account is to make sure you have the email address associated with those commits added to your account.</p>
<p>You can check which email address you’ve been using either by appending <code>.patch</code> to the end of a commit url or you can check your git config:</p>
<pre><code> git config --list 
</code></pre>
<p>If you want to remove your real name from your settings, you can update that too.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/using-git/setting-your-username-in-git" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/using-git/setting-your-username-in-git" target="_blank" rel="noopener">Setting your username in Git - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/using-git/setting-your-username-in-git" target="_blank" rel="noopener">//docs.github.com/en/github/using-git/setting-your-username-in-git</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
